internal/cli/atlas/accesslogs/list.go

```diff
@@ -32,8 +32,8 @@ import (
 const (
 	success      = "success"
 	fail         = "fail"
-	listTemplate = `HOSTNAME	CLUSTER NAME	AUTH RESULT	LOG LINE {{range .AccessLogs}}
-{{if .Hostname}}{{.Hostname}} {{else}}N/A{{end}}{{.Hostname}}	{{if .ClusterName}}{{.ClusterName}} {{else}}N/A{{end}}	{{.AuthResult}}	{{.LogLine}}{{end}}
+	listTemplate = `HOSTNAME	AUTH RESULT	LOG LINE {{range .AccessLogs}}
```

Comment:
[q] why cluster name is not shown anymore?

Reply:
Ahhhh my bad I should have added this here: https://jira.mongodb.org/browse/CLOUDP-167913

Reply:
oh got it! thanks